Zilliqa/README.md

```diff
@@ -121,29 +116,34 @@ Proxy contract is a relay contract that redirects calls to it to the token contr
 
 | Name | Description & Privileges |
 |--|--|
-|`init_admin` | The initial admin of the contract which is usually the creator of the contract.  `init_admin` is also the initial value of `admin`. |
-|`admin` | Current `admin` of the contract initialized to `init_admin`. Certain critical actions can only be performed by the `admin`, e.g., changing the current implementation of the token contract. |
-|`initiator`| The user who calls the proxy contract that in turns call the token contract. After deployment, the address of the proxy contract will be made known to the user and the code will be visible directly from the block explorer. |
+|`init_admin`| The initial admin of the contract. It is usually the creator of the contract. |
```

Comment:
Description of admin seems to have disappeared, i.e. changing implementation of contract

Reply:
done